packages/observablehq-compiler/src/compiler.ts

```diff
@@ -128,86 +200,43 @@ function createImportVariable(name?: string, alias?: string) {
 }
 type ImportVariableFunc = ReturnType<typeof createImportVariable>;
 
-async function createModule(parsed: ParsedImportCell, text: string, baseUrl: string) {
-    const otherModule = isRelativePath(parsed.src) ?
-        await importFile(parsed.src, baseUrl) :
-        await importCompiledNotebook(parsed.src);
-
-    const importVariables: ImportVariableFunc[] = [];
-    const variables: VariableFunc[] = [];
-    parsed.specifiers.forEach(spec => {
-        const viewof = spec.view ? "viewof " : "";
-        importVariables.push(createImportVariable(viewof + spec.name, viewof + spec.alias));
-        if (spec.view) {
-            importVariables.push(createImportVariable(spec.name, spec.alias));
-        }
-    });
-    variables.push(createVariable(true, undefined, ["md"], md => {
-        return md`\`\`\`JavaScript
-${text}
-\`\`\``;
-    }));
-
-    const retVal = (runtime: ohq.Runtime, main: ohq.Module, inspector?: ohq.InspectorFactory) => {
-
-        let mod = runtime.module(otherModule);
-        if (parsed.injections.length) {
-            mod = mod.derive(parsed.injections, main);
-        }
-        variables.forEach(v => v(main, inspector));
-        importVariables.forEach(v => v(main, mod));
-        return mod;
-    };
-    retVal.importVariables = importVariables;
-    retVal.variables = variables;
-    retVal.dispose = () => {
-        importVariables.forEach(v => v.dispose());
-        variables.forEach(v => v.dispose());
-        otherModule.dispose();
-    };
-    retVal.write = (w: Writer) => {
-        otherModule.write(w);
-        w.importDefine(parsed);
-    };
-    return retVal;
-}
-type ModuleFunc = Awaited<ReturnType<typeof createModule>>;
-
-async function createCell(node: ohq.Node, baseUrl: string) {
+// Cell  ---
+async function createCell(node: ohq.Node, options: CompileOptions) {
+    node.id = "" + node.id;
```

Comment:
Should this type just be a string?

Reply:
This is no longer needed and removed, thx.